docs/general/ad-filtering/create-own-filters.md

```diff
@@ -250,20 +250,34 @@ Example:
 
 ### Basic modifiers {#basic-rules-basic-modifiers}
 
+The following modifiers are the most simple and frequently used.
+Basically, they just limit the scope of rule application.
+
 <!-- Please keep them sorted -->
 
-- [`$app`](#app-modifier)
-- [`$denyallow`](#denyallow-modifier)
-- [`$domain`](#domain-modifier)
-- [`$header`](#header-modifier)
-- [`$important`](#important-modifier)
-- [`$match-case`](#match-case-modifier)
-- [`$method`](#method-modifier)
-- [`$popup`](#popup-modifier)
-- [`$third-party`](#third-party-modifier)
-- [`$to`](#to-modifier)
+| Modifier \ Products | [CL apps][cl-apps] | [ExtChr][ext-chr] | [ExtFf][ext-ff] | [iOS app][ios-app] | [Safari][ext-saf] | [CB][and-cb] |
+| --- | :---: | :---: | :---: | :---: | :---: | :---: |
+| [$app](#app-modifier) | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| [$denyallow](#denyallow-modifier) | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| [$domain](#domain-modifier) | ✅ | ✅ | ✅ | ✅ [*](#domain-modifier-limitations) | ✅ [*](#domain-modifier-limitations) | ✅ |
+| [$header](#header-modifier) | ✅ | ⏳ | ⏳ | ❌ | ❌ | ❌ |
+| [$important](#important-modifier) | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| [$match-case](#match-case-modifier) | ✅ | ✅ | ✅ | ? | ? | ✅ |
+| [$method](#method-modifier) | ⏳ | ✅ | ✅ | ❌ | ❌ | ❌ |
+| [$popup](#popup-modifier) | ✅ * | ✅ | ✅ | ✅ * | ✅ * | ? |
+| [$third-party](#third-party-modifier) | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| [$to](#to-modifier) | ⏳ | ✅ | ✅ | ❌ | ❌ | ❌ |
+
+:::note
 
-The following modifiers are the most simple and frequently used. Basically, they just limit the scope of rule application.
+- ✅ — fully supported
+- ✅ * — supported, but reliability may vary or limitations may occur; check the modifier description for more details
+<!-- - 🧩 — may already be implemented in nightly or beta versions but is not yet supported in release versions -->
```

Comment:
it will be challenging to maintain this in an up-to-date state :)

Reply:
moreover, some platforms may have fresh modifiers in release tracks but others - only in pre-release ones

Reply:
@slavaleleka what about to add new columns to the secret table, and synchronize syntax changes into it first, and then from it to AGLint and KB?
Or add a new page with all these tables, and use that file as primary synchronization source?

Reply:
synchronisation sounds great but seems to be a separate task